### DISCUSSION TITLE
Characters - Full face, Body type etc!

### DISCUSSION THREAD

**Discussion by MaaiG** [FakeFishGames/Barotrauma#7735] [Ideas and suggestions]
![2021-12-12 14 53 47 ru wikipedia org 88a6639955a3](https://user-images.githubusercontent.com/93348499/145711111-4e7f0355-3c32-4e7e-942f-79e268ef582f.png)

All this gives more way for creativity (including mods)!

- **1. Add "full-face".**
In the game, character visualization is only in "profil".
The character will can stand up facing or  his back , fix devices etc.
Ultimately, the character can shown in profile, face or back to the object with which it interacts depending on how depth the object is located.
![2021-12-12 15 18 03 www pinterest ru 856c33c411e3](https://user-images.githubusercontent.com/93348499/145712346-5da6eb98-f6e7-49f1-bbc9-a1a325927e5e.png)
**Or more**
![Image 001_02-16](https://user-images.githubusercontent.com/93348499/145904200-13d0e4e6-d7a9-48cd-8bf5-f481ea56a8ac.png)

- **2. Sometimes characters don’t do anything. Why don’t they change positions?**
![2021-12-14 02 09 41 www pinterest com mx d285686479bd](https://user-images.githubusercontent.com/93348499/145903817-9389c732-70d2-4ced-8170-d9534fe27686.png)

- **3. Add "body type".**
![Image 001_15-26](https://user-images.githubusercontent.com/93348499/145712381-e0c8a7a4-299e-49f8-b148-abac8da7e9da.png)

- **Move on!**
![Без названия (26)](https://user-images.githubusercontent.com/93348499/145713176-24208028-a9fe-4c6e-832b-5bc1970f5b42.jpg)

**Comment by StruggleDuck** [comment]
I think the reason behind not having something like this is that the game is 2D, and something like this wouldn't really fit the theme of the game. It would also be a lot of work to implement.

**Comment by MaaiG** [comment]
>I think the reason behind not having something like this is that the game is 2D...

For 2d [it is not a problem](https://www.moddb.com/mods/brutal-doom/videos/brutal-doom-v20-official-trailer#imagebox)! 2d - does not prevent the character from standing face or back. You can draw as many sprites as you want.

I would say that 2D has no problem at all: https://www.youtube.com/watch?v=20gKpcirZDE
Nowadays, problems are mostly in 3D. )) Too powerful computers - no need to optimize or invent and that's bad!

**Reply by WickedWolfy** [reply]
The issue would be the sheer amount of clothing sprites that would need to be made for all body types.
Issue = no, Immense time consumption = yes.

**Reply by MaaiG** [reply]
Something like that... "Nothing is impossible to a willing heart" :)

**Comment by Udrakan** [comment]
This would be a nightmare to do in a ragdoll-based physics game, not to mention breaking the style it's going for...

**Comment by Bebrich** [comment]
Having a character rotate his head towards the camera is nightmare fuel. What is he looking at? Is he looking at you? Cases of schizophrenia will skyrocket. Our economy cannot support this. Weird request.

Might make for a good creepypasta, but not a good addition to the actual game.

Also it would be hell to implement, unless they do change characters to 3d. Every single piece of equipment, times every single bodytype, times every single angle type of effort. Would take like a couple of years of constant drawing to do this.
